doc/build/dependency.md

```diff
@@ -0,0 +1,94 @@
+# Building dependencies
```

Comment:
I updated it and added a reference to this file.

> For example, with the suggested NDK_DIR the script doesn't find the build toolchain python script.

Are you sure? Are you using the latest NDK?